packages/trigger-pager/src/triggerPager.test.ts

```diff
@@ -69,12 +126,12 @@ describe('triggerPager', () => {
         },
       }
     })
-    // @ts-expect-error
-    api.mockReturnValue({ post })
 
-    await triggerPager({
-      title: '',
-      description: '',
-    })
+    assert.doesNotReject(async () =>
+      triggerPager({
```

Comment:
```suggestion
    await assert.doesNotReject(async () =>
      await triggerPager({
```
or simply
```suggestion
    await triggerPager({
```

Reply:
Isn't `await`-ing a Promise returned from an async function a bit of an anti-pattern? 

I'm going to go for the first option as I like to be explicit with the assert here. Also communicates intent better I think. I had to read the code of `triggerPager` before I understood what Julian was doing here.

Reply:
It seems there has been some new discoveries about that, and the ESLint rule was deprecated, you can read the reasons in https://eslint.org/docs/latest/rules/no-return-await.

Not there's also this possibility:

```suggestion
    await assert.doesNotReject(triggerPager({